README.md

```diff
@@ -1,13 +1,86 @@
 # [BaniDB](https://pypi.org/user/KhalisFoundation/)
-### _Python package for BaniDB API_
-
 [![](bdb.svg)](http://banidb.com)
 
-### Installation
-- Run [`pip install banidb`](https://pypi.org/project/banidb/) in your root directory
-
 # Vision Statement
 
 BaniDB's vision is to create a single, universally accessible Gurbani Database for Sikh websites and applications. BaniDB is, and will continue to be, the most accurate and complete Gurbani database ever created for use by Sikhs around the world.
 
 In order to make this vision possible, members of this collaborative effort work to ensure that the platform is selfsustaining, tested, and secure.
+
+## Python package for BaniDB API
+
+### Installation
+With pip
+```
+pip install banidb
+```
+
+### Usage
+Quick Start
+```
+shabad = banidb.random()
+print(shabad)
+
+```
+Search shabad by first letters"
+```
+print("\tSearching Bandhana Har Bandhana ....")
+shabad = banidb.search("bhbgggr")
+print(shabad)
+```
+

```

Comment:
Shall we add all the available functions and uses to make it more user-friendly?

Reply:
Yes, we should. Would you like to create an issue to track that, and then add it?

Reply:
I can merge this one as is, if you like this approach?

Reply:
Ya, that would be great 👍🏽